core/templates/dev/head/pages/exploration_player/RefresherExplorationConfirmationModalService.js

```diff
@@ -35,7 +35,21 @@ oppia.factory('RefresherExplorationConfirmationModalService', [
             function($scope, $uibModalInstance, $window) {
               $scope.confirmRedirect = function() {
                 redirectConfirmationCallback();
-                var url = '/explore/' + refresherExplorationId;
+                var collectionId = UrlService.getUrlParams().collection_id;
+                var parentIdList = UrlService.getQueryFieldValuesAsList(
+                  'parent');
+                var urlTemplate = '/explore/<refresher_exploration_id>';
+                var interpolationValues = {};
+                interpolationValues.refresher_exploration_id =
+                  refresherExplorationId;
+                url = UrlInterpolationService.interpolateUrl(
+                  urlTemplate,interpolationValues);
```

Comment:
Add space after comma

(I'm a bit surprised our eslint isn't catching all this stuff, we should investigate why.)

Reply:
Done, yeah, it caught the no space between infix operators but not this.